File: source codes/Sasha_Angermueller_Lab_3/Coins.cs
using System;

class Coins {
  public static void Main (string[] args) {
    int NQuarters, NDimes, NNickles, NPennies, Dollars, Cents;
    Console.WriteLine ("Enter Number of Quarters: ");
    NQuarters = int.Parse(Console.ReadLine());
    Console.WriteLine ("Enter Number of Dimes: ");
    NDimes = int.Parse(Console.ReadLine());
    Console.WriteLine ("Enter Number of Nickles: ");
    NNickles = int.Parse(Console.ReadLine());
    Console.WriteLine ("Enter Number of Pennies: ");
    NPennies = int.Parse(Console.ReadLine());
    Console.WriteLine();
    Console.WriteLine ("You entered " + NQuarters + " Quarters");
    Console.WriteLine ("You entered " + NDimes + " Dimes");
    Console.WriteLine ("You entered " + NNickles + " Nickles");
    Console.WriteLine ("You entered " + NPennies + " Pennies");
    Console.WriteLine();
    NQuarters = NQuarters * 25;
    NDimes = NDimes * 10;
    NNickles = NNickles * 5;
    NPennies = NPennies * 1;
    Dollars = (NQuarters + NDimes + NNickles + NPennies)/100;
    Cents = (NQuarters + NDimes + NNickles + NPennies)%100;
    Console.WriteLine ("Your total is " + Dollars + " Dollars and " + Cents + " Cents.");
    Console.ReadLine();
  }
}

File: School source codes/Lab 3/Coins.rb
puts "Enter Number of Quarters: "
NQ = gets.chomp.to_i
puts "Enter Number of Dimes: "
ND = gets.chomp.to_i
puts "Enter Number of Nickles: "
NN = gets.chomp.to_i
puts "Enter Number of Pennies: "
NP = gets.chomp.to_i
puts "\nYou entered #{NQ} Quarters"
puts "You entered #{ND} Dimes"
puts "You entered #{NN} Nickles"
puts "You entered #{NP} Pennies\n\n"
NQuarters = NQ * 25
NDimes = ND * 10
NNickles = NN * 5
NPennies = NP * 1
Dollars = (NQuarters + NDimes + NNickles + NPennies) / 100
Cents = (NQuarters + NDimes + NNickles + NPennies) % 100
puts "Your total is #{Dollars} Dollars and #{Cents} Cents."